libsolidity/formal/CHC.cpp

```diff
@@ -1210,6 +1206,16 @@ void CHC::addVerificationTarget(
 	smtutil::Expression _errorId
 )
 {
+	solAssert(m_currentContract || m_currentFunction, "");
+	SourceUnit const* source = nullptr;
+	if (m_currentContract)
+		source = sourceUnitContaining(*m_currentContract);
+	else
+		source = sourceUnitContaining(*m_currentFunction);
+	solAssert(source, "");
+	if (!source->annotation().experimentalFeatures.count(ExperimentalFeature::SMTChecker))
+		return;
```

Comment:
These were added so that verification targets created for imported code do not get checked if that SourceUnit doesn't have the pragma too.